packaging/rpm/xpra-html5.spec

```diff
@@ -55,8 +55,13 @@ or by any other web server.
 %setup
 
 %install
-mkdir -p %{buildroot}/usr/share/xpra/www
-%{python} ./setup.py install %{buildroot}/usr/share/xpra/www/ %{minifier}
+mkdir -p %{buildroot}%{_datadir}/xpra/www
+mkdir -p %{buildroot}%{_sysconfdir}/xpra/html5-client
+%{python} ./setup.py install %{buildroot}%{_datadir}/xpra/www/ %{minifier}
+# Move and symlink configuration files
+cp %{buildroot}%{_datadir}/xpra/www/default-settings.txt %{buildroot}%{_sysconfdir}/xpra/html5-client/
+rm %{buildroot}%{_datadir}/xpra/www/default-settings.txt
+ln -s %{buildroot}%{_sysconfdir}/xpra/html5-client/default-settings.txt %{buildroot}%{_datadir}/xpra/www/default-settings.txt
```

Comment:
It should be the other way around.
One could point `apache` or `nginx` to `/usr/share/xpra/www` and those web servers usually don't follow symlinks whereas `vi` / `nano` / whatever do.

Also I still think that the symlink logic should be in just one place called by `./setup.py install` instead of duplicating it in 3 places.

Reply:
> It should be the other way around.

You mean you still want editable file in `/uar/share` ?

Reply:
Ah, I see.